Documentation/BuildInstructionsMacOS.md

```diff
@@ -22,6 +22,9 @@ Toolchain/BuildFuseExt2.sh
 
 # (option 2) genext2fs
 brew install genext2fs
+
+# for kernel debugging, on Apple Silicon
```

Comment:
(aarch64 is not yet fully operational… but maybe you could indicate (pretty obvious thing) that this is only for `x86_64` builds too? 🤔)